### PR TITLE
Make empty views consistent

### PR DESCRIPTION
**Pull request by agzuniverse**
Changes the font size of "No recent conversations" in ConversationsCard to match that of SearchEmptyState

### CODE REVIEW

**Comment by borisyankov** [comment]
Maybe even better - extract the style and reuse?

**Comment by agzuniverse** [comment]
Done.

File: src/conversations/ConversationsCard.js [hunk 1]
```diff
@@ -82,7 +80,9 @@ class ConversationsCard extends PureComponent<Props> {
           />
         </View>
         {conversations.length === 0 ? (
-          <Label style={componentStyles.emptySlate} text="No recent conversations" />
+          <View style={styles.emptyStateContainer}>
```
Comment (jainkuniya) [inline src/conversations/ConversationsCard.js]: Is this extra wrapper required?
Can't we do it with `Label` itself?

Comment (agzuniverse) [reply src/conversations/ConversationsCard.js]: Label does not return text in a view, it just returns a `Text` element so the wrapper is required.

Comment (borisyankov) [reply src/conversations/ConversationsCard.js]: Why do you need a wrapper though?

Comment (agzuniverse) [reply src/conversations/ConversationsCard.js]: To align using `justifyContent`.
Also `SearchEmtpyState` uses a wrapper View, and I moved the styles of that view to miscStyles and reused it here.

**Pull request closed without merging**